server/controllers/applicant/updateApplicant.js

```diff
@@ -0,0 +1,43 @@
+const Applicant = require('../../database/models/applicant');
+const validationUpdate = require('../../utils/validation/applicantValidation');
+
+const updateApplicant = (req, res) => {
+  const { body } = req;
+  const { id } = req.params;
```

Comment:
@ha77an If you want, you could combine these two lines into one line:
`const { body, { params: id }} = req;`